libs/zzz/ui/src/Character/card/CharacterCardContent.tsx

```
@@ -0,0 +1,96 @@
+  const charStat = getCharStat(characterKey)
+  const chKey = character?.key
+  const cProfession = charStat?.specialty
+  const cAttribute = charStat?.attribute
+  const characterLevel = character?.level
+  const cRarity = charStat?.rarity
+  return (
+    <>
+      <Box display="flex" gap={1.5} alignItems="center">
+        <ImgIcon size={2} src={rarityDefIcon(cRarity)} />
+        <Typography variant="h4">{chKey}</Typography>
+        <ElementIcon ele={cAttribute}></ElementIcon>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)